vars.tf

```diff
@@ -13,11 +13,17 @@ variable "vpc_cidr" {
   default     = "172.20.0.0/16"
   description = "CIDR range for the VPC?"
   validation {
-    condition     = cidrsubnet(var.vpc_cidr, 3, 5) != ""
+    condition     = cidrsubnet(var.vpc_cidr, 3, 5 + var.subnet_cidrs_offset) != ""
```

Comment:
Hmmm, this kind of validation is not possible. Only the current variable can be used. Any thoughts on how to achieve the validation correctly?